modules/loki-deployment-sizing.adoc

```diff
@@ -7,35 +7,14 @@
 
 Sizing for Loki follows the format of `N<x>._<size>_` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
 
-[NOTE]
-====
-1x.extra-small is for demo purposes only, and is not supported.
-====
-
 .Loki Sizing
 [options="header"]
 |========================================================================================
-|                              | 1x.extra-small  | 1x.small            | 1x.medium
-| *Data transfer*              | Demo use only.  | 500GB/day           | 2TB/day
-| *Queries per second (QPS)*   | Demo use only.  | 25-50 QPS at 200ms  | 25-75 QPS at 200ms
-| *Replication factor*         | None            | 2                   | 3
-| *Total CPU requests*         | 5 vCPUs         | 36 vCPUs            | 54 vCPUs
-| *Total Memory requests*      | 7.5Gi           | 63Gi                | 139Gi
-| *Total Disk requests*        | 150Gi           | 300Gi               | 450Gi
+|                              | 1x.extra-small     | 1x.small            | 1x.medium
+| *Data transfer*              | 500GB/day          | 500GB/day           | 2TB/day
+| *Queries per second (QPS)*   | 1-25 QPS at 200ms  | 25-50 QPS at 200ms  | 25-75 QPS at 200ms
```

Comment:
@periklis I didn't have any value for this in the Jira so I went with this as a placeholder guess. Please confirm an actual QPS value.